src/stratus-wps/pom.xml

```diff
@@ -34,11 +34,19 @@
             <artifactId>s3-stream-upload</artifactId>
             <version>1.0.1</version>
         </dependency>
+
         <dependency>
             <groupId>org.geotools.xsd</groupId>
             <artifactId>gt-xsd-gml3</artifactId>
             <version>${geotools.version}</version>
         </dependency>
+        <!--
+        <dependency>
+            <groupId>org.geoserver</groupId>
+            <artifactId>gs-wps-spatialstatistics</artifactId>
+            <version>${geoserver.version}</version>
+        </dependency>
+        -->
```

Comment:
Same comment as above, just delete this.

Reply:
done